packages/mdc-checkbox/mdc-checkbox.scss

```diff
@@ -23,18 +23,21 @@
 
 .mdc-checkbox {
   @include mdc-checkbox-base_;
-  @include mdc-checkbox-ink-color($mdc-checkbox-mark-color);
-  @include mdc-checkbox-focus-indicator-color($mdc-checkbox-baseline-theme-color);
   @include mdc-ripple-surface;
   @include mdc-states($mdc-checkbox-baseline-theme-color);
   @include mdc-ripple-radius($mdc-checkbox-ripple-radius);
-  @include mdc-checkbox-container-colors();
 
   @include mdc-theme-dark {
     @include mdc-checkbox-container-colors($unmarked-stroke-color: $mdc-checkbox-border-color-dark);
   }
 }
 
+@at-root {
```

Comment:
I'll be darned! I thought you could only `@include` mixins in Sass if you were already inside a CSS selector (even if the mixin itself has a CSS selector in it).

But I guess Sass treats `@at-root` like a selector, so ... awesome!

Reply:
WEeeee!